auction-app-frontend/src/components/BiddersTable/BiddersTable.jsx

```diff
@@ -0,0 +1,100 @@
+import { useEffect, useState } from "react";
+import moment from "moment";
+
+import { findBidsByProductId } from "src/services/bidService";
+import { BIDDERS_TABLE_ROWS, BIDDERS_TABLE_DEFAULT_PAGE_SIZE } from "src/constants";
+import { BUTTON_LABELS, BUTTON_VARIANTS } from "src/constants";
+import { userProfilePicture } from "src/assets/images";
+
+import { Button, ButtonLoadingIndicator, ErrorComponent } from "src/components";
+
+import "./style.scss";
+
+const BiddersTable = ({ productId }) => {
+    const [loading, setLoading] = useState();
+    const [error, setError] = useState(null);
+    const [biddersList, setBiddersList] = useState([]);
+    const [page, setPage] = useState(0);
+    const [size, setSize] = useState(BIDDERS_TABLE_DEFAULT_PAGE_SIZE);
+    const [hasMore, setHasMore] = useState(true);
+
+    const fetchBiddersList = () => {
+        setLoading(true);
+
+        findBidsByProductId(productId, page, size)
+            .then(response => {
+                setBiddersList((prevItems) => 
+                    page === 0 
+                        ? [...response.content]
+                        : [...prevItems, ...response.content]
+                );
+
+                setHasMore(response.last !== true);
+            })
+            .catch(error => {
+                setError(error);
+                setLoading(false);
+            })
+            .finally(() => {
+                setLoading(false);
+            });
+    };
+
+    useEffect(() => {
+        if(productId) fetchBiddersList();
+    }, [productId, page]);
+
+    const handlePageChange = (page) => {
+        setPage((prevPage) => prevPage + 1);
+    }
+
+    const formatDate = (dateString) => {
+        return moment(dateString, "YYYY-MM-DD").format("D MMMM YYYY");
+    };
+
+    const formatProfilePicture = (profilePicture) => {
+        return profilePicture || userProfilePicture;
+    };
+
+    if (error) return <ErrorComponent message={ error } />;
+
+    return (
+        <div className="bidders-table-container body-semibold">
+        <table>
```

Comment:
This could be a nice thing to do (you don't have to do it now, but maybe later if you want it to improve this project)
Create components for table and table fields, to avoid styling that repeats in different components, because all tables look the same